### ISSUE TITLE
'Save as screenshot' sometimes doesn't export screenshot

### ISSUE THREAD

**Issue by shiftyscales** [Frooxius/NeosPublic#1142]
Reported by H3BO3#7957 and Lucaedr#4319, 'Save as screenshot' will apparently sometimes not save/export the file properly. I'll add more details as I learn of them.

**Comment by shiftyscales** [comment]
An exception found in a log provided by H3BO3:
```
10:51:22 PM.650 ( 35 FPS)	Exception running asynchronous task:
System.AggregateException: One or more errors occurred. ---> System.NullReferenceException: Object reference not set to an instance of an object
  at FrooxEngine.PhotoMetadata+<<NotifyOfScreenshot>b__14_0>d.MoveNext () [0x000b9] in <c0b323f9dcf94fad96116b8edd552530>:0 
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.NullReferenceException: Object reference not set to an instance of an object
  at FrooxEngine.PhotoMetadata+<<NotifyOfScreenshot>b__14_0>d.MoveNext () [0x000b9] in <c0b323f9dcf94fad96116b8edd552530>:0 <---


   at void BaseX.UniLog.Error(string message, bool stackTrace)
   at void FrooxEngine.CoroutineManager.CheckExceptions(Task task)
   at void System.Threading.Tasks.ContinuationTaskFromTask.InnerInvoke()
   at void System.Threading.Tasks.Task.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at bool System.Threading.ThreadPoolWorkQueue.Dispatch()
   at bool System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()
10:51:56 PM.511 ( 34 FPS)	Exception running asynchronous task:
System.AggregateException: One or more errors occurred. ---> System.NullReferenceException: Object reference not set to an instance of an object
  at FrooxEngine.PhotoMetadata+<<NotifyOfScreenshot>b__14_0>d.MoveNext () [0x000b9] in <c0b323f9dcf94fad96116b8edd552530>:0 
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.NullReferenceException: Object reference not set to an instance of an object
  at FrooxEngine.PhotoMetadata+<<NotifyOfScreenshot>b__14_0>d.MoveNext () [0x000b9] in <c0b323f9dcf94fad96116b8edd552530>:0 <---


   at void BaseX.UniLog.Error(string message, bool stackTrace)
   at void FrooxEngine.CoroutineManager.CheckExceptions(Task task)
   at void System.Threading.Tasks.ContinuationTaskFromTask.InnerInvoke()
   at void System.Threading.Tasks.Task.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at bool System.Threading.ThreadPoolWorkQueue.Dispatch()
   at bool System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()
```

**Comment by Frooxius** [comment]
Should be fixed in 2020.10.14.462, thanks!
